android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-photo-view')
```

Comment:
@andrewda In this dependence there is the "pinch to zoom" feature, it's convenient. This may come in handy in the future (to display other images with zoom). Although I agree with you, as long as this is not necessary, why can use the usual component `Image`?